lib/rules/no-return-await.js

```diff
@@ -44,7 +46,32 @@ module.exports = {
             context.report({
                 node: context.getSourceCode().getFirstToken(node),
                 loc: node.loc,
-                messageId: "redundantUseOfAwait"
+                messageId: "redundantUseOfAwait",
+                suggest: [
+                    {
+                        messageId: "removeAwait",
+                        fix(fixer) {
+                            const areAwaitAndAwaitedExpressionOnTheSameLine = node.loc.start.line === node.argument.loc.start.line;
+
+                            if (!areAwaitAndAwaitedExpressionOnTheSameLine) {
+                                return null;
+                            }
+
+                            const sourceCodeText = context.getSourceCode().text;
+
+                            const startOfAwait = node.range[0];
+                            const endOfAwait = startOfAwait + "await".length;
```

Comment:
We prefer getting indexes from nodes/tokens. If we have `awaitToken` (per my previous comment), we can do something like this:

```js
const [startOfAwait, endOfAwait] = awaitToken.range;
```

Reply:
Thanks! I'll adjust it after the decision above is concluded.

Reply:
Done